src/modules/product/controller/productController.ts

```diff
@@ -387,6 +387,16 @@ const buyerViewWishList = async (req: ExtendRequest, res: Response) => {
   }
 };
 
+const buyerReviewProduct = async (req: ExtendRequest, res: Response) => {
+  const { rating, feedback } = req.body
+  const productId = req.params.id
+  const createdReview = await productRepositories.userCreateReview({ rating, feedback, productId, userId: req.user.id })
+  return res.status(httpStatus.OK).json({
+    message: "Product reviewed successfully",
+    data: createdReview

```

Comment:
Please follow the conventional code format we are using to return the response data.